src/sass/_side_panel.scss

```diff
@@ -1,5 +1,26 @@
 .is-overlay {
+  // mobile
+  background-color: #fff;
   height: 100vh;
+  padding: 0 calc($sp-unit * 3);
+  padding-bottom: calc($sp-unit * 1.5);
+  padding-top: $sp-unit;
   position: absolute;
+  right: 0;
+  width: 100vw;
   z-index: 103 !important;
+
+  .p-panel__header {
+    padding: 0;
+  }
+
+  .panel-footer {
+    hr {
+      margin-bottom: calc($sp-unit * 1.5);
+    }
+  }
+}
+
+.panel-notification {
+  padding: 0;
```

Comment:
Can we use `u-no-padding` instead?

Reply:
sure, but that means we would have to specify `u-no-padding` on every `NotificationRow` we place inside a panel. I thought this is an easier approach